File: src/main/java/org/jcodec/containers/mp4/DataBoxes.java
package org.jcodec.containers.mp4;

import org.jcodec.containers.mp4.boxes.AliasBox;
import org.jcodec.containers.mp4.boxes.UrlBox;

public class DataBoxes extends Boxes {
    public DataBoxes() {
        mappings.put(UrlBox.fourcc(), UrlBox.class);
        mappings.put(AliasBox.fourcc(), AliasBox.class);
        mappings.put("cios", AliasBox.class);
    }
}

File: src/main/java/org/jcodec/containers/mp4/DataBoxes.kt
package org.jcodec.containers.mp4

import org.jcodec.containers.mp4.boxes.AliasBox
import org.jcodec.containers.mp4.boxes.UrlBox

class DataBoxes : Boxes() {
    init {
        mappings[UrlBox.fourcc()] = UrlBox::class.java
        mappings[AliasBox.fourcc()] = AliasBox::class.java
        mappings["cios"] = AliasBox::class.java
    }
}